### DISCUSSION TITLE
Avram::Params.new({"name" => "value"}) failed on newest main branch

### DISCUSSION THREAD

**Discussion by zw963** [luckyframework/lucky#1713] [Q&A] [answered]
I am working a kemal + avarm project, i use avram main branch. (4b1cb28fa704db693dee8503700be13c73611dc5)

I try to introduce Avram::Params, make my params more safely, but it failed with following error:

```
 > 20 |   params = Avram::Params.new({"name" => "billy"})
                                 ^--
Error: no overload matches 'Avram::Params.new' with type Hash(String, String)

Overloads are:
 - Avram::Params.new(hash : ::Hash(::String, ::Array(::String)))
 - Avram::Params.new()
```

Yes, i saw probably we are refactoring this feature, old spec example not works anymore.

So, how to do that? why we change from [old](https://github.com/luckyframework/avram/blob/v0.23.0/spec/avram/operations/save_operation_spec.cr#L391) usage to this?

Thank you.

**Comment by jwoertink** [comment, marked as answer]
With the next release, in order to support Arrays in params, I had to change this. Now you do

```crystal
params = Avram::Params.new({"name" => ["billy"]})
```
